Gems/Atom/RHI/Vulkan/Code/Source/RHI/FrameGraphCompiler.h

```diff
@@ -100,6 +101,28 @@ namespace AZ
             VkPipelineStageFlags srcPipelineStageFlags = prevScopeAttachment ? GetResourcePipelineStateFlags(*prevScopeAttachment) : VK_PIPELINE_STAGE_BOTTOM_OF_PIPE_BIT;
             VkAccessFlags srcAccessFlags = prevScopeAttachment ? GetResourceAccessFlags(*prevScopeAttachment) : 0;
 
+            // this makes vkCmdClearColorImage is apart of this resource barrier
```

Comment:
The comment is a little confusing. Maybe change it to the following->
"Add VK_ACCESS_TRANSFER_WRITE_BIT in case we want to do a clear operation."